.gitlab/image_build/docker_linux.yml

```
@@ -13,7 +13,7 @@
     - DOCKER_REGISTRY_LOGIN=$(aws ssm get-parameter --region us-east-1 --name ci.datadog-agent.$DOCKER_REGISTRY_LOGIN_SSM_KEY --with-decryption --query "Parameter.Value" --out text)
     - aws ssm get-parameter --region us-east-1 --name ci.datadog-agent.$DOCKER_REGISTRY_PWD_SSM_KEY --with-decryption --query "Parameter.Value" --out text | docker login --username "$DOCKER_REGISTRY_LOGIN" --password-stdin "$DOCKER_REGISTRY_URL"
     # Pull base images
-    - inv -e docker.pull-base-images --signed-pull $BUILD_CONTEXT/$ARCH/Dockerfile
+    - inv -e docker.pull-base-images $BUILD_CONTEXT/$ARCH/Dockerfile --no-content-trust # currently --no-content-trust as ubuntu:21.10 is not properly signed on DockerHub
```

I’m not sure it was needed to add a `--no-content-trust` flag redundant with `--signed-pull`.
By default `--signed-pull` is true, so we need to explicitly specify `--no-signed-pull`.

```suggestion
    - inv -e docker.pull-base-images --no-signed-pull $BUILD_CONTEXT/$ARCH/Dockerfile --no-content-trust # currently --no-signed-pull as ubuntu:21.10 is not properly signed on DockerHub
```